### PR TITLE
Add linked template support to README

### PR DESCRIPTION
**Pull request by MarcusFelling**
#1225

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +152,76 @@ Select a new parameter file or create a new parameter file to update the associa
...
+
+```json
+        {
+            "name": "linkedDeployment1",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/StephenWeatherford/template-examples/master/linkedTemplates/uri/child.json"
```
Comment (StephenWeatherford) [inline README.md:192]: Let's find a link from the quick starts repo (yes, I'll update the changelog too).
Also, I think it should use expressions.  Perhaps the _artifactionsLocation pattern.

File: README.md [hunk 2]
```diff
@@ -151,6 +152,76 @@ Select a new parameter file or create a new parameter file to update the associa
...
+                },
+                "parameters": {
+                }
+            }
+        }
+```
+
+- Relative to deployed template location
+
+```json
```
Comment (StephenWeatherford) [inline README.md:202]: Does github support jsonc so it doesn't show red on the comments?

File: README.md [hunk 3]
```diff
@@ -151,6 +152,76 @@ Select a new parameter file or create a new parameter file to update the associa
...
+            }
+        }
+```
+
+Additional features:
+
+- ["Light-bulb" and snippet support](https://github.com/microsoft/vscode-azurearmtools/blob/mafellin/1225-linked-template-README/README.md#add-missing-parameters) to fill in parameter values for a linked template
+- CTRL-click on relativePath value or click on code lens to navigate to linked template
+
 ## Template navigation
```
Comment (StephenWeatherford) [inline README.md:225]: We should discuss the full validation issue, or point to a wiki page that does.

File: README.md [hunk 4]
```diff
@@ -151,6 +152,76 @@ Select a new parameter file or create a new parameter file to update the associa
 
 ![Image showing the parameter file creation options (none, new, and browse)](./images/undo-change-mapping-two.png)
 
+## Linked Template Support
+
+When a [linked template](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/linked-templates#linked-template) is referenced, schema and parameter validations are provided.
```
Comment (StephenWeatherford) [inline README.md:157]: It's not just parameters that are validated.  Some semantics of the linked template are validated, using the parameter values that were passed in.

File: README.md [hunk 5]
```diff
@@ -151,6 +152,76 @@ Select a new parameter file or create a new parameter file to update the associa
 
 ![Image showing the parameter file creation options (none, new, and browse)](./images/undo-change-mapping-two.png)
 
+## Linked Template Support
```
Comment (StephenWeatherford) [inline README.md:155]: Images/animations?

File: README.md (hunk 2 above)
Comment (MarcusFelling) [reply README.md:202]: Looks like JSON5 works

File: README.md (hunk 3 above)
Comment (MarcusFelling) [reply README.md:225]: ok, can you assist with a wiki page? I'm not familiar with the history behind it.

File: README.md (hunk 5 above)
Comment (MarcusFelling) [reply README.md:155]: Added animation for param quick fix. Can't figure out how to re-size images for screenshots on GitHub. I'll give it another shot tomorrow.

**Review by StephenWeatherford — APPROVED** [review]

**Pull request merged**